algo/src/main/java/com/hartwig/actin/algo/evaluation/tumor/TumorUtil.java

```diff
@@ -0,0 +1,32 @@
+package com.hartwig.actin.algo.evaluation.tumor;
+
+import com.hartwig.actin.algo.datamodel.Evaluation;
+import com.hartwig.actin.algo.datamodel.EvaluationResult;
+import com.hartwig.actin.algo.datamodel.ImmutableEvaluation;
+import com.hartwig.actin.algo.evaluation.EvaluationFactory;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class TumorUtil {
+    @NotNull
+    public static Evaluation evaluateBooleanMetastasis(@Nullable Boolean hasMetastases,
```

Comment:
Are we testing this via the HasXxxMetastases tests? Another approach could be to unit test this code directly and invert the dependency such that we inject a non-static implementation of this into each rule

Reply:
Yes, I noticed that each of those evaluation functions was doing exactly the same thing so this is a simple refactor and the existing `Has*Metastases` tests still apply.

Do you think non-static usage of the util would be an improvement here? I'm still calibrating my functional/OO leanings after switching back from Scala.

I like that we currently have tests that verify and document each specific evaluation function regardless of whether they make use of a shared utility. It wouldn't hurt to test the `TumorUtil` in addition to each evaluation function though.

Reply:
I looked into doing this but the only thing that varies between evaluation criteria is the String representing the metastasis type, so I can't think of a test we'd want to add now that would make use of injection. I'm happy with just reducing duplication for now but we can refactor further if we find more use cases for the `TumorUtil`.